src/components/NewCommentForm/NewCommentForm.tsx

```diff
@@ -1,39 +1,92 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './NewCommentForm.scss';
+import { createComment } from '../../api/comments';
 
-export const NewCommentForm: React.FC = () => (
-  <form className="NewCommentForm">
-    <div className="form-field">
-      <input
-        type="text"
-        name="name"
-        placeholder="Your name"
-        className="NewCommentForm__input"
-      />
-    </div>
-
-    <div className="form-field">
-      <input
-        type="text"
-        name="email"
-        placeholder="Your email"
-        className="NewCommentForm__input"
-      />
-    </div>
-
-    <div className="form-field">
-      <textarea
-        name="body"
-        placeholder="Type comment here"
-        className="NewCommentForm__input"
-      />
-    </div>
-
-    <button
-      type="submit"
-      className="NewCommentForm__submit-button button"
+type Props = {
+  postId: number | null
+  comments: IComment[] | null
+  setComments: (comment: IComment[] | null) => void
+};
+
+export const NewCommentForm: React.FC<Props> = ({
+  postId,
+  comments,
+  setComments,
+}) => {
+  const [name, setName] = useState('');
+  const [email, setEmail] = useState('');
+  const [body, setBody] = useState('');
+
+  const addNewComment = (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+
+    const addComment: IComment = {
+      postId,
+      id: comments && comments.length + 1,
+      name,
+      email,
+      body,
+
+    };
+
+    createComment(addComment);
```

Comment:
you should make the request to server for getting new array of comments after adding the new one